cmd/govim/testdata/rename.txt

```diff
@@ -3,7 +3,8 @@
 vim ex 'e main.go'
 errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
 vim ex 'call cursor(6,7)'
-vim ex 'call execute(\"GOVIMRename banana\")'
+vim ex 'call execute(\"GOVIMRename\")'
```

Comment:
@dragfire - yes, searching for the "right" thing here is tricky. What you actually need to do here is use `feedkeys`, which allows you to simulate user input, i.e. the keys the user pressed. Here is an example from the completion tests:

https://github.com/govim/govim/blob/1391094d768940d3d55c05db9b77592bee976e1b/cmd/govim/testdata/complete.txt#L7

Reply:
Note the content within the single quotes is escaped because it is used directly as a JSON value in the special `vim` commands.

Reply:
@myitcv what makes using it difficult is that `vim ex 'call execute(\"GOVIMRename\")'` after executing this test is blocked waiting for user input and even if I use `feedkeys` in the next line, it will never get executed. That' what I noticed while I was running the test.

Reply:
![Screen Shot 2019-09-13 at 9 37 38 AM](https://user-images.githubusercontent.com/6277978/64866871-478dc080-d60a-11e9-9c74-461d9c844c82.png)
It is stuck forever after calling rename.

Reply:
Ah I see. You want to replace `vim ex 'call execute(\"GOVIMRename\")'` with something like (untested) `vim ex 'call feedkeys(\":GOVIMRename\\<ESC>\", \"xt\")'`. i.e. you need to simulate (via `feedkeys`) the running of the command as well as the `<Esc>`

Reply:
I changed it to `vim ex 'call feedkeys(\":GOVIMRename\\<ESC>\", \"xt\")'` because we need to press enter first. So same problem as above.

Reply:
But have you made any changes to support the handling of escape? I can't see those in this PR unless I'm missing something?

Reply:
I wanted to see this test execute first https://github.com/govim/govim/pull/530/files#diff-9ec4d0a996ae226a61329bdb8ace6cb1R26 
I was thinking if this works it should print that log and it will be much easier to write the code after that.

Reply:
Ah, I see. Yes, that's a good thought. I'll see if I can find time to help out over the weekend.

Reply:
Just a quick thought: if you currently hit escape, doesn't that throw an error? I'm not sure what will happen in a test in that case. Might be worth trying to do a feedkeys with `<Esc>` where you know it will not fail and see whether the test still blocks. You can always "test" out `feedkeys` calls in Vim itself using `:call feedkeys(":GOVIMRename\<CR>\<Esc>", "xt")` (note you do not need the escaping here)